check_source_in_factory.py

```diff
@@ -148,7 +148,14 @@ def _check_requests(self, project, package, rev):
                         return True
                     elif req.state.name == 'review':
                         self.logger.info("request still in review")
-                        return None
+                        for r in req.reviews:
+                            if r.by_project and r.state == 'new' and r.by_project.startswith('openSUSE:Factory:Staging:'):
```

Comment:
the request will never be 'new' AND pending staging review - then it's still state == review